### PR TITLE
Naïve Modal component

### PR DESCRIPTION
**Pull request by sapegin**

### CODE REVIEW

File: src/exercises/7-Naive_Modal/Propmpt.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Input from '../../components/primitives/Input';
+import Button from '../../components/primitives/Button';
+import Heading from '../../components/primitives/Heading';
+import Stack from '../../components/layout/Stack';
```
Comment (okonet) [inline src/exercises/7-Naive_Modal/Propmpt.js]: Lets not use any of those and just HTML since that's what we expect from participants.

Comment (sapegin) [reply src/exercises/7-Naive_Modal/Propmpt.js]: Didn't we agreed on introducing primitives with a short exercise _before_ this one?

File: src/exercises/6-Dialog/Dialog.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+				<label>
+					<h2>{message}</h2>
+					<input
+						type="text"
+						value={value}
+						onChange={event => setValue(event.target.value)}
+					/>
+				</label>
+				<div>
+					<button type="submit" variant="primary">
```
Comment (okonet) [inline src/exercises/6-Dialog/Dialog.js]: Variant? Let’s remove

**Comment by sapegin** [comment]
I did some cleanup, added more text and added templates. Going to merge now, so I can continue with the next exercise. Feel free to comment though — I'll fix in the next pull request.

**Pull request merged**